src/build/supportedExtensions.js

```diff
@@ -48,10 +47,11 @@ exports.extensions = {
     { icon: 'config', extensions: ['makefile', '.env.example'], filename: true },
     { icon: 'compass', extensions: [] },
     { icon: 'composer', extensions: ['composer.json', 'composer.lock'], filename: true },
-    { icon: 'csharp', extensions: ['cs', 'csx'], languages: [languages.csharp] },
+    { icon: 'cpp', extensions: ['hpp', 'cc', 'cxx'], languages: [languages.cpp] },
+    { icon: 'csharp', extensions: ['csx'], languages: [languages.csharp] },
     { icon: 'cshtml', extensions: ['cshtml'] },
     { icon: 'csproj', extensions: ['csproj'] },
-    { icon: 'css', extensions: ['css'], languages: [languages.css] },
+    { icon: 'css', extensions: [], languages: [languages.css] },
```

Comment:
It stays there for 2 reasons:
- in case there is another extension that has the same icon.
- schema consistency.